website/source/docs/providers/do/r/loadbalancer.html.markdown

```diff
@@ -0,0 +1,98 @@
+---
+layout: "digitalocean"
+page_title: "DigitalOcean: digitalocean_loadbalancer"
+sidebar_current: "docs-do-resource-loadbalancer"
+description: |-
+  Provides a DigitalOcean Loadbalancer resource. This can be used to create, modify, and delete Loadbalancers.
+---
+
+# digitalocean\_loadbalancer
+
+Provides a DigitalOcean Loadbalancer resource. This can be used to create,
```

Comment:
With the exception of the resource name (`digitalocean_loadbalancer`) I think "Load Balancer" should be two words